graphbolt/src/fused_csc_sampling_graph.cc

```diff
@@ -1693,7 +1721,7 @@ std::enable_if_t<is_labor(S), int64_t> Pick(
           probs_or_mask.value(), picked_data_ptr);
     } else {
       int64_t picked_count;
-      AT_DISPATCH_FLOATING_TYPES(
+      GRAPHBOLT_DISPATCH_ALL_TYPES(
```

Comment:
LaborPick supports dispatch to all existing torch scalar types when I use this macro @Rhett-Ying.